etc/systemd/aquilon-broker.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Quattor Aquilon broker
+After=network.target
+
+[Service]
+EnvironmentFile=/etc/sysconfig/aqd
+# Update python path to reflect your configuration. If it is in a virtualenv it will do whatever is
```

Comment:
this is an example to for a `virtenv.conf` file to put in `/etc/systemd/aquilon.service.d` (or generated with `ncm-systemd`); but should not be in the main unit
if the variable have no defaults, they are also not really allowed. the shipped unitfile should work on a system deployed by eg rpms, this looks like customisation to me (but i do not now if the variables can have defaults, eg by setting them with `Environment`

Reply:
I don't think we can define variable defaults. They have to exist, this is also documented (or should be!) in the installation documentation. I have no problem putting the standard python path as the default and updating the comment for a virtualenv.